src/appleseed/renderer/kernel/rendering/tilecallbackbase.h

```diff
@@ -64,7 +64,9 @@ class TileCallbackBase
     void on_tile_begin(
         const Frame*            frame,
         const size_t            tile_x,
-        const size_t            tile_y) override
+        const size_t            tile_y,
+        const size_t            thread_index = -1,
```

Comment:
Why assign default values? Moreover, `-1` will be converted to an unsigned value (`~0`) so if default values are really needed, I would rather use `~0`.

Reply:
Remove default values as explained in #dev.